src/ERC20Helper.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: AGPL-3.0-only
-pragma solidity ^0.8.7;
+pragma solidity 0.8.7;
```

Comment:
We should not enforce solc version in libraries. The inherited contracts will enforce the solc version. We should not have to upgrade libraries or abstract contracts just so that new inheritors can work.